tutorials/llm/llama-3/pruning-distillation/03_b_width_pruning.ipynb

```diff
@@ -0,0 +1,92 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8bc99d2f-9ac6-40c2-b072-12b6cb7b9aca",
+   "metadata": {},
+   "source": [
+    "### Step 3: Prune the finetuned-teacher model to create a student\n",
+    "In the second method, we will width-prune. In width-pruning, we trim the neurons, attention heads and embedding channels. \n",
```

Comment:
fix punctuation

  "In the second method, we will width-prune. In width-pruning, we trim the neurons, attention heads, and embedding channels. \n",